editor/src/components/inspector/flex-section.tsx

```diff
@@ -106,10 +122,80 @@ export const FlexSection = React.memo(() => {
     [addFlexLayoutSystem, addGridLayoutSystem],
   )
 
+  const grid = useEditorState(
+    Substores.metadata,
+    (store) =>
+      layoutSystem === 'grid' && store.editor.selectedViews.length === 1
+        ? MetadataUtils.findElementByElementPath(
+            store.editor.jsxMetadata,
+            store.editor.selectedViews[0],
+          )
+        : null,
+    'FlexSection grid',
+  )
+
+  const columns: GridCSSNumber[] = React.useMemo(() => {
+    if (grid == null) {
+      return []
+    }
+
+    const fromProps =
+      grid.specialSizeMeasurements.containerGridPropertiesFromProps.gridTemplateColumns
+    const calculated = grid.specialSizeMeasurements.containerGridProperties.gridTemplateColumns
+
+    if (fromProps?.type !== 'DIMENSIONS' && calculated?.type !== 'DIMENSIONS') {
+      return []
+    }
+
+    let dimensions: GridCSSNumber[] = []
+
+    if (calculated?.type === 'DIMENSIONS') {
+      dimensions.push(...calculated.dimensions)
+    }
```

Comment:
nitpick, but I think this simple if does not warrant using a let a mutate, and copying the array. I would just use

`const dimensions = calculated?.type === 'DIMENSIONS' ? calculated.dimensions : null`

Reply:
https://github.com/concrete-utopia/utopia/pull/6118/commits/b7cfd566e9c4781df314c0c90f8f92e70c8b2e67